modules/rostests/apitests/user32/CharFuncs.c

```diff
@@ -0,0 +1,597 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Tests for Char* functions
+ * COPYRIGHT:   Copyright 2022 Stanislav Motylkov <x86corez@gmail.com>
+ */
+
+#include "precomp.h"
+
+#include <strsafe.h>
+#include <windows.h>
```

Comment:
Which other functions need the whole windows.h ? Maybe you only need winnls.h or something similar?

Reply:
`windows.h` is for `WideCharToMultiByte`.

Reply:
OK then use `#include <winnls.h>` instead.